geode-core/src/test/java/org/apache/geode/test/dunit/standalone/DUnitLauncher.java

```diff
@@ -146,6 +146,15 @@ private static boolean isHydra() {
     }
   }
 
+  /**
+   * Launch DUnit. If the unit test was launched through the hydra framework, leave the test alone.
+   */
+  public static void launchIfNeeded(int vmCount) {
```

Comment:
Not sure why this shows up in this pr.  I think this change already made it into develop a few weeks ago